game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/ToolBoxWindow.java

```diff
@@ -0,0 +1,97 @@
+package games.strategy.engine.lobby.moderator.toolbox;
+
+import java.awt.Component;
+import java.net.URI;
+import java.util.Optional;
+
+import javax.swing.JFrame;
+
+import org.triplea.http.client.moderator.toolbox.ModeratorToolboxClient;
+import org.triplea.swing.JFrameBuilder;
+import org.triplea.swing.JPanelBuilder;
+import org.triplea.swing.SwingComponents;
+
+import games.strategy.engine.lobby.client.login.LobbyPropertyFetcherConfiguration;
+import games.strategy.engine.lobby.client.login.LobbyServerProperties;
+import games.strategy.engine.lobby.moderator.toolbox.tabs.TabFactory;
+import games.strategy.triplea.settings.ClientSetting;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+
+/**
+ * This window shows a series of tabs that provide CRUD operations to a moderator. Each tab roughly maps
+ * to a DB table.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class ToolBoxWindow {
+
+  // TODO: remove this main method, just here for demo purposes.
+  public static void main(final String[] args) {
```

Comment:
Is this still being used?

Reply:
In the short-term, yes. It'll probably be removed when some of the last tabs are added.

I'm thinking before moderator toolbox can be considered 'done', we'll want:
- moderators tab for super-moderator to be able to issue API keys
- thinking we may want an API keys tab, not 100% sure yet
- a users tab for moderators to do bans
- a bans tab to view bans and remove bans

So there is still work to be done and the temp main method should remain useful for a little bit yet.